lib/hammer_cli_katello/host_collection_erratum.rb

```diff
@@ -15,6 +15,12 @@ class InstallCommand < HammerCLIKatello::HostCollection::InstallContentBaseComma
         :format => HammerCLI::Options::Normalizers::List.new,
         :attribute_name => :content)
 
+      def execute
+        warn "This command uses katello agent and will be removed in favor of remote execution in a future release"
+        warn "The new command is `hammer job-invocation create --feature katello_errata_install`. Specify the host collection with the --search-query parameter"
```

Comment:
How do I specify a host collection using `--search-query`? Can an example search query be added here?

Reply:
it can be written `--search-query='host_collection = MyCollection'` or `--search-query='host_collection_id = 1`

I'm slightly hesitant to be this explicit because I don't want it to be interpreted as "oh, I can just copy and paste this example and expect it to work", if that makes sense..

Reply:
I didn't know the correct syntax (e.g. that I could use `host_collection = ` or `host_collection_id = `) so I'm just looking for some way for this message to provide that hint.

Reply:
something like:
```
Specify the host collection with the --search-query parameter, e.g. `--search-query 'host_collection = MyCollection'` or `--search-query 'host_collection_id=6'`